pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Problem
+- *Description of why you made this PR*
+
+## Solution
+- *And how do you fix that problem*
+
+## PR Status
+- [ ] Code finished.
+- [ ] Tested with Package_check.
+- [ ] Fix or enhancement tested.
+- [ ] Upgrade from last version tested.
+- [ ] Can be reviewed and tested.
+
+## Validation
+---
+- [ ] **Code review**
+- [ ] **Approval (LGTM)**  
+*Code review and approval have to be from a member of @YunoHost/apps group*
```

Comment:
This app isn't a High Quality app, no need to ask to the Apps group for each PR.

Reply:
removed